.github/workflows/extension_ci.yml

```diff
@@ -29,13 +29,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        pg: [12, 13, 14, 15, 16]
+        pg: [12, 13, 14, 15, 16, 17]
+        partman: [4.7.3, 5.1.0]
+        exclude:
+          # pg_partman does not support pg < 14
+          - pg: 12
+            partman: 5.1.0
+          - pg: 13
+            partman: 5.1.0
     container: pgxn/pgxn-tools
     steps:
       - run: pg-start ${{ matrix.pg }}
```

Comment:
I suggest you add the pg_partman version to this line.

Reply:
Do you mean pass the pg_partman version in as an argument to `pg-start`?

Reply:
Sorry, I meant the `name` line:

``` yaml
    name: Test 🐘 ${{ matrix.pg }} with pg_partman ${{ matrix.partman }}
```

Reply:
ahh yes! makes sense, good call.

Reply:
nice 😎 

![image](https://github.com/user-attachments/assets/c57a9c47-1552-424e-9f47-8598dbad466b)